tests/integ/modin/frame/test_applymap.py

```diff
@@ -28,25 +28,33 @@
 )
 
 
+@pytest.fixture(params=["applymap", "map"])
+def method(request):
+    """
+    how keyword to pass to merge.
+    """
+    return request.param
+
+
 @pytest.mark.parametrize("data,func,return_type", BASIC_DATA_FUNC_RETURN_TYPE_MAP)
 @sql_count_checker(query_count=7, udf_count=1)
-def test_applymap_basic_without_type_hints(data, func, return_type):
+def test_applymap_basic_without_type_hints(data, func, return_type, method):
```

Comment:
Can we also have a test to check for raising NotImplementedError when `na_action=ignore`?

Reply:
Done